app/jobs/sync_all_users_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SyncAllUsersJob < CronJob
+  self.cron_expression = "0 0 * * *"
+
+  queue_as :sync_users
```

Comment:
What's this doing?

Reply:
https://edgeguides.rubyonrails.org/active_job_basics.html#queues

In short, there can be many queues for our jobs. Say, one for very high priority stuff, one for sending emails, one for syncing users. And you can make those queues have different priorities - so that our worker, which is executing the queues, goes for the important things first.

We don't have many queues and prioritising in place yet, but we probably will.